xLights/effects/assist/SketchCanvasPanel.cpp

```diff
@@ -836,7 +835,7 @@ void SketchCanvasPanel::setBackgroundBitmap(std::unique_ptr<wxBitmap> bm)
 
 void SketchCanvasPanel::ClosePath()
 {
-    if( !m_pathClosed ) {
+    if (!m_pathClosed) {
```

Comment:
The main change was the piece highlights by Scott .. a change to the file picker to use the bulkedit file picker.
Understand about the code changes but being new, I was relying on the clang-format to make sure I was doing it correctly. I have changed the vs to not to the whole file but it will still hit the some of the code.
Maybe everyone needs to chime in on the clang-format to ensure everyone is okay with what is in there?
Where can those discussions take place? I really want to see if we can whittle down the 650+ tickets to get to the meat of the needs out there.

Reply:
Do we want to change update the clang format, currently it is:
SpaceBeforeParens: true
SpacesInParentheses: false

Reply:
> Do we want to change update the clang format, currently it is: SpaceBeforeParens: true SpacesInParentheses: false

You posted while I was typing.  For that one probably don't want to change it because I'm likely the only one that liked it that way.  I just had a SW lead early on that insisted on that way and I ended up kinda liking how your eyes focus on what's in the middle.  But I can adapt to whatever.

Reply:
yes visual studio is enforcing the .clang-format rules committed 3 years ago. The idea being that with that in place, the code will always have the same style but does take people to buy in to setting the rules and applying them :-) 
I assumed (wrongly) the code was already using those rules but it appears that as I hit various files, it was not used universally.
In my prior job that was an auto code review reject if the code didn't pass the linter - man that took some getting used to.

Reply:
By default VS only formats the code you are adding not the whole file. If you run format document or have a commit hook it would format the whole file

Reply:
Thoughts on this one .. is it worth me going back to try and create the code::blocks  version for this file?
The original author did not use/create code::blocks.